server/middleware/token/utils/createClientJWT.js

```diff
@@ -0,0 +1,17 @@
+const { sign } = require("jsonwebtoken");
+require("dotenv").config();
+
+const createTokens = (user) => {
+  const { email, userId } = user;
+  // console.log("ged" + email);
+  // console.log("ged" + userId);
+  const accessToken = sign(
+    { email: email, userId: userId },
+    process.env.JWT_SECRET,
+    { expiresIn: "30d" }
```

Comment:
```suggestion
    { expiresIn: "1d" }
```
make the expiration to be no longer than a day

Reply:
updated to 1 day